client/signup/steps/store-features/index.tsx

```diff
@@ -118,7 +118,7 @@ export default function StoreFeaturesStep( props: Props ): React.ReactNode {
 
 					<p>
 						{ translate(
-							'If you have multiple products or require extensive order and shipping management than this might suit your needs better.'
+							'If you have multiple products or require extensive order and shipping management then this might suit your needs better.'
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.